### PR TITLE
static/redirect: redirect all packages to their own godoc

### PR DESCRIPTION
**Pull request by kortschak**
This is a more complete (but still not ideal) solution to godoc redirection.

Please take a look.

Updates #4.

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -1,5 +1,137 @@
-# Redirect attempts to get to docs to a more helpful location.
-/v1/gonum/*	https://pkg.go.dev/gonum.org/v1/gonum/	302
-/v1/plot/*	https://pkg.go.dev/gonum.org/v1/plot/	302
-/v1/netlib/*	https://pkg.go.dev/gonum.org/v1/netlib/	302
-/v1/exp/*	https://pkg.go.dev/gonum.org/v1/exp/	302
\ No newline at end of file
+# Generated by generate_directs.sh; DO NOT EDIT.
+# Redirects to get to docs to a more helpful location.
+/v1/gonum/graph/formats/dot/internal/parser/* https://pkg.go.dev/gonum.org/v1/gonum/graph/formats/dot/internal/parser/ 302
```
Comment (sbinet) [inline static/_redirects]: couldn't we sort those? (mainly to have a stable output)
probably injecting a `| sort` would do.

Comment (sbinet) [reply static/_redirects]: ah... no. well not *as simple* as just adding a dumb `|sort`...

Comment (kortschak) [reply static/_redirects]: They are sorted, by length of line. The redirect matcher works first found, so this makes choices specific.

**Comment by kortschak** [comment]
I don't understand why Netlify doesn't like having go.mod entries for exp (I thought it was because of absence of license, but this is not the case since netlib also caused the problem — perhaps v0.0.0 version?). We can deal with it later.

File: static/_redirects (hunk 1 above)
Comment (kortschak) [reply static/_redirects]: Making the sort, `sort -k1,1nr -k2` would make it completely stable.

Comment (kortschak) [reply static/_redirects]: Done

**Review by sbinet — APPROVED** [review]

**Pull request merged**